source/wasm/build.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Copyright (c) 2020 The Khronos Group Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+NUM_CORES=$(nproc)
+echo "Detected $NUM_CORES cores for building"
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+VERSION=$(sed -n '0,/^v20/ s/^v\(20[0-9.]*\).*/\1/p' $DIR/../../CHANGES).${GITHUB_RUN_NUMBER:-0}
```

Comment:
This ignores any `-dev` suffix which may be present.
But npm assumes semantic versioning.  (https://docs.npmjs.com/about-semantic-versioning )

So we could have:
    

           CHANGES file says                npm sees
           2021.5-dev                             2021.5.##
           2021.5                                     2021.5.##  (## incremeents by built number)
           2021.6-dev                             2021.6.#
           2021.6                                     2021.6

In principle, breaking changes could occur in the -dev series, which could break assumptions downstream.
On the other hand, the current JS interface is very narrow: assemble and disassemble, and the environment enums.   So there's essentially no risk this will break.